A3.md

```diff
@@ -0,0 +1,93 @@
+Channel Tracing
+----
+* Author(s): Noah Eisen (ncteisen@google.com)
+* Approver: markdroth, ctiller
+* Status: Draft
+* Implemented in: N/A
+* Last updated: 01/25/17
+* Discussion at: https://groups.google.com/forum/#!topic/grpc-io/WFDj3KeHYTI
+
+## Abstract
+
+The document proposes adding a method to the gRPC C-core Api called grpc_channel_get_trace() which returns a detailed trace for the channel in the form of a JSON string. Eventually this trace could be hooked up to a server with a clean UI, and will be instrumental in debugging channel related bugs.
```

Comment:
If this gRFC is intended to cover a mechanism that can be implemented in all languages, then let's remove references to C-core and to particular function names from everything except the "Implementation" section.